cpp/qsn_qsn.h

```diff
@@ -50,11 +50,11 @@ inline Str* UEscape(int codepoint) {
 }
 #else
 inline Str* XEscape(Str* ch) {
-  assert(0);
+  NotImplemented(); // Unused
 }
 
 inline Str* UEscape(int codepoint) {
-  assert(0);
+  NotImplemented(); // Unused
```

Comment:
I just added this as part of the GC translation, so that's why it's hidden behind !USING_OLD_QSN.

There is also USING_OLD_MYLIB.  So we need to work out a good scheme for porting to the GC runtime.

It is technically unused now but will be a different implementation for the GC.

The reason is that the existing functions just leak memory!

Reply:
Sounds good.  I assume for a fair number of the `Unused` tags this is the case